content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
```

Comment:
security perspective, this ~default~ behavior is not recommended ~because~ as it allows a compromised webhook ~could then~ to conduct

Reply:
The current one is more accurate.